worlds/pokemon_rb/regions.py

```diff
@@ -2344,6 +2351,7 @@ def search_for_exit(entrance, region, checked_regions):
                             return found_exit
                 return None
 
+            e = multiworld.get_entrance("Underground Path Route 5 to Underground Path North South", 1)
```

Comment:
This was the cause of the Lingo error
```suggestion
            e = multiworld.get_entrance("Underground Path Route 5 to Underground Path North South", player)
```

Reply:
Oh yeah, that would do it :D